playbooks/roles/certbot/tasks/main.yml

```diff
@@ -54,6 +61,33 @@
     dest: /etc/letsencrypt/renewal-hooks/deploy/restart-nginx.sh
     mode: 0755
 
+# The systemtd certbot renew timer that's included in the certbot snap package does not send an email on failure.
+# For this reason we disable the timer and use a custom cronfile with a MAILTO instead.
+- name: Disable systemd timer
+  systemd:
+    name: snap.certbot.renew.timer
+    state: stopped
+    enabled: no
+
+# Custom cron job for renewing certificates. The only difference from the official snap systemd timer is
+# that this cron job will send an email to ops if any of the certificates fail to renew.
+- name: Create cron job to periodically renew certificates
+  cron:
+    name: certbot renew
+    user: root
+    job: "perl -e 'sleep int(rand(43200))' && /snap/bin/certbot -q renew"
+    minute: '0'
+    hour: '*/12'
+    cron_file: certbot-renew
+
+- name: Set MAILTO variable for certbot renew cron job
```

Comment:
Have we tested to make sure this works? I'm not able to see the ops@ email. Can we force a failure?

Reply:
@Kelketek, we can force a failure by removing the DNS records for an instance whose certificates we want to try renewing.

Reply:
@Kelketek, @mtyaka, I tried to force a renewal failure by removing the DNS records for the domains in a certificate and attempted to force renew it. But that didn't work because certbot is caching the domain validations from a server for a certificate for 60 days. So I could only check and verify that emails are sent to the `MAILTO` address for any stdout/stderr messages in command output.

Reply:
I did get an email to `MAILTO` when I was still testing with my own email (before I switched to ops@). It was not exactly because certificate renewal failed, but it was because the cryptography library printed some python 3.5 incompatibility warnings.

Reply:
@Kelketek @lgp171188 If you're subscribed to the ops list, you should have received an email alert with the subject "[ops] Cron <root@cert-manager-stage> perl -e 'sleep int(rand(43200))' && /snap/bin/certbot -q renew" a few minutes ago.

Reply:
@mtyaka, I am not subscribed to that list. But if you are and have received the email, that is good enough for me 👍🏼